src/storage/double_write_buffer.c

```diff
@@ -737,61 +737,53 @@ dwb_adjust_write_buffer_values (unsigned int *p_double_write_buffer_size, unsign
   min_size = DWB_MIN_SIZE;
   max_size = DWB_MAX_SIZE;
 
+  assert (IS_POWER_OF_2 (min_size) && IS_POWER_OF_2 (max_size));
```

Comment:
is this assertion required? 
min_size and max_size are initialized using constant values ​​that satisfy the 'assert' condition.

And I think this min/max_size variable refers to the size of the buffer, not the number of blocks. So the assert should check if it is a multiple of 512 * 1024.